app/app.js

```diff
@@ -35,9 +37,18 @@ app.use((err, req, res, next) => {
       .send({ message: "Unprocessable Entity - request must be a number" });
   } else next(err);
 });
+app.use((err, req, res, next) => {
+  if (err.code === "23503" && err.constraint === "comments_author_fkey") {
+    res.status(400).send({ message: "Bad Request - Username does not exist" });
+  } else if (
+    err.code === "23503" &&
+    err.constraint === "comments_review_id_fkey"
+  ) {
+    res.status(404).send({ message: "Not Found - review_id does not exist" });
+  } else next(err);
+});
 
 app.use((err, req, res, next) => {
-  console.log(err);
```

Comment:
you want to keep this specific console.log there to see what is eventually ending up there